src/NzbDrone.Core/MetadataSource/SkyHook/SkyHookProxy.cs

```diff
@@ -443,14 +443,14 @@ private static SeriesStatusType MapSeriesStatus(string status)
             return SeriesStatusType.Continuing;
         }
 
-        private static Ratings MapRatings(RatingResource rating)
+        private static Core.Music.Ratings MapRatings(RatingResource rating)
```

Comment:
It is imported, but I had to differentiate between core.tv.ratings and core.music.ratings. Once we can purge the core.tv dependence we can get rid of this.